docs/guides/lido-tokens-integration-guide.md

```diff
@@ -23,6 +28,18 @@ Lido's stTokens are widely adopted across the Ethereum ecosystem:
 - There are live ChainLink [stETH/USD](https://app.ens.domains/name/steth-usd.data.eth) and [stETH/ETH](https://etherscan.io/address/0x86392dC19c0b719886221c78AB11eb8Cf5c52812) price feeds on Ethereum.
 - There are live ChainLink stETH/USD price feeds on [Arbitrum](https://docs.chain.link/docs/arbitrum-price-feeds/) and [Optimism](https://docs.chain.link/docs/optimism-price-feeds/). These also have ChainLink wstETH-stETH exchange rate data feeds.
 
+### LDO
+
+[LDO](#ldo-1) is a Lido governance ERC-20 compliant token derived from the [MiniMe Token](https://github.com/Giveth/minime).
+The latter allows the LDO balance to be tracked and retrieved for the arbitrary block number, which is an essential part of the
+Lido voting mechanics.
+
+### unstETH
+
+A non-fungible token (NFT) is used to represent a withdrawal request position [inside the in-protocol withdrawals queue](/contracts/withdrawal-queue-erc721) when a stToken holder decides to redeem it for Ether via the protocol.
+
+Lido's [unstETH](#withdrawals-unsteth) is non-fungible and implements the ERC-721 token standard in contrast to the abovementioned tokens.
```

Comment:
"In contrast" implies stark opposing differences. Here, however, we're just pointing to the token implementing a different standard, not necessarily opposing. I would say something like,
"Unlike stETH, wstETH, and LDO, Lido's unstETH is nonfungible and implements the ERC-721 token standard."